store/config.go

```diff
@@ -100,16 +108,31 @@ func (c Config) String() string {
 		c.EthGasPriceDefault.String(),
 		c.LinkContractAddress,
 		c.MinimumContractPayment.String(),
+		oracleContractAddress,
 	)
 }
 
 func parseEnv(cfg interface{}) error {
 	return env.ParseWithFuncs(cfg, env.CustomParsers{
-		reflect.TypeOf(big.Int{}):  bigIntParser,
-		reflect.TypeOf(LogLevel{}): levelParser,
+		reflect.TypeOf(&common.Address{}): addressParser,
+		reflect.TypeOf(big.Int{}):         bigIntParser,
+		reflect.TypeOf(LogLevel{}):        levelParser,
 	})
 }
 
+func addressParser(str string) (interface{}, error) {
```

Comment:
Does this need to be an empty interface? Looks like you make it `*common.Address`.

Reply:
Yeah, otherwise it won't conform to the interface at the use site:

```go
cannot use addressParser (type func(string) (*common.Address, error)) as type env.ParserFunc in map value
```